meson.build

```diff
@@ -1483,13 +1482,32 @@ vaapi_libplacebo = {
 }
 if vaapi_libplacebo['use']
     features += vaapi_libplacebo['name']
-    sources += files('video/out/hwdec/hwdec_vaapi_pl.c')
 endif
 
 if vaapi_egl['use'] or vaapi_libplacebo['use']
     sources += files('video/out/hwdec/hwdec_vaapi.c')
 endif
 
+dmabuf_interop_gl = {
+    'name': 'dmabuf_interop_gl',
+    'use': vaapi_egl['use']
```

Comment:
should probably depend on `drm && egl`

and since we already have drm as a dep that'd also mean you can include `drm_fourcc.h` without worry